src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'bookStore/categories/CHECK_STATUS';
+
+const initialState = [];
+
+export default function categoriesReducer(state = initialState, action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+    default:
+      return state;
+  }
+}
+
+export function checkStatus() {
+  return {
+    type: CHECK_STATUS,
+  };
```

Comment:
[OPTIONAL] I would suggest you  use ES6 syntax for all your function `(arrow function)`